pkg/release-controller/release.go

```diff
@@ -520,3 +521,64 @@ func GetImageInfo(releaseInfo ReleaseInfo, architecture, pullSpec string) (*imag
 	}
 	return &config, nil
 }
+
+func GetVerificationJobs(rcCache *lru.Cache, eventRecorder record.EventRecorder, lister *MultiImageStreamLister, release *Release, releaseTag *imagev1.TagReference, artSuffix string) (map[string]ReleaseVerification, error) {
+	if release.Config.As != ReleaseConfigModeStable {
+		return release.Config.Verify, nil
+	}
+	jobs := make(map[string]ReleaseVerification)
+	for k, v := range release.Config.Verify {
+		jobs[k] = v
+	}
+	version, err := SemverParseTolerant(releaseTag.Name)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get full stable verification jobs: %w", err)
+	}
+	isName := fmt.Sprintf("%d.%d%s", version.Major, version.Minor, artSuffix)
+	klog.Infof("Getting imagestream %s/%s", release.Target.Namespace, isName)
+	isLister := lister.ImageStreams(release.Target.Namespace)
+	if isLister == nil {
+		return nil, fmt.Errorf("failed to get imagestream %s", release.Target.Namespace)
+	}
+	imageStream, err := isLister.Get(isName)
+	if err != nil {
+		// TODO: make this an error once the artSuffix flag has been added to all commands
+		//return nil, fmt.Errorf("failed to get imagestream %s/%s: %w", release.Target.Namespace, isName, err)
+		return release.Config.Verify, nil
+	}
+	versionedRelease, ok, err := ReleaseDefinition(imageStream, rcCache, eventRecorder, *lister)
+	if err != nil {
+		return nil, err
+	} else if !ok {
+		return nil, fmt.Errorf("failed to get release definition from stream %s/%s", release.Target.Namespace, isName)
+	}
```

Comment:
Are you sure this logic works correctly when the `--art-suffix` isn't set?  It looks like you're potentially crossing the streams...
Say a new 4.11 release gets published into `ocp/release`.  This logic looks like it would interrogate the `ocp/4.11` imagestream's Release Configuration, which technically belongs to `CI`.

Reply:
Updated the code so that it returns just the stable imagestream verification if the artSuffix is empty